tools/config-generator/main_test.go

```diff
@@ -14,17 +14,13 @@ limitations under the License.
 package main
 
 import (
-	"os"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 )
 
-func TestMain(m *testing.M) {
```

Comment:
why?

Reply:
TestMain does setup only once for the package, not for each individual test as I believed. See this discussion: https://github.com/golang/go/issues/27927

Reply:
oh, ha forgot about that, I thought it was a test called Main... dur